frontend/src/setupProxy.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+module.exports = function(app) {
+  app.use(
+    '/plugins',
+    createProxyMiddleware({
+      target: 'http://localhost:4654',
+      changeOrigin: true,
+    })
+  );
+};
```

Comment:
In case of config we use the apiProxy from the k8s lib, we require the above because we want to proxy "when in dev mode all requests to /plugins to the backend" this is just a development related change, we did similar thing in Nebraska as well for /api and /config